From 41e1462b378d770474cd7d82eb64a8635d196bfb Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Tue, 3 Aug 2021 00:00:30 -0700
Subject: [PATCH 216/230] Editor: Disable magnifier position animation

In my opinion, this animation serves no purpose because it doesn't
correspond to any animations in the text selection/handle flow, and is
barely noticeable in typical text selection cases. In addition, it makes
the magnifier feel slow/laggy when selecting a large block of text
quickly.

Change-Id: I5fed9fc06db4491dc2811815741830001d295030
---
 core/java/android/widget/Editor.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/widget/Editor.java b/core/java/android/widget/Editor.java
index 49d456b5323..e492f659dfa 100644
--- a/core/java/android/widget/Editor.java
+++ b/core/java/android/widget/Editor.java
@@ -4576,7 +4576,7 @@ public class Editor {
          * animation is added to the jump.
          */
         private void show(final float x, final float y) {
-            final boolean startNewAnimation = mMagnifierIsShowing && y != mLastY;
+            final boolean startNewAnimation = false;
 
             if (startNewAnimation) {
                 if (mAnimator.isRunning()) {
-- 
2.25.1

